composer.json

```diff
@@ -12,9 +12,9 @@
   ],
   "require": {
     "php": ">=7.3.0",
```

Comment:
You should then also increase php version to `>=7.4.0`. Because 7.3.* versions don't support typed properties, which you used in your PR.

Reply:
Done